lib/wallet/account.js

```diff
@@ -702,7 +720,13 @@ class Account extends bio.Struct {
    */
 
   changeAddress() {
-    const key = this.changeKey();
+    let key;
+    if (this.staticAddress) {
+      // branch 1 -> changes
+      key = this.getSpecificKey(0, 1);
```

Comment:
We want the change addr to be static too, I'll push a commit to replace this with 0, 0 if there's nothing else

Reply:
it's already static and we have tests for it to verify the parameter 1 on the getSpecificKey is for branch (1 -> changes)
am i missing sth ?